docs/standard/assembly/snippets/identify/csharp/ExamplePeReader.cs

```diff
@@ -0,0 +1,58 @@
+﻿//<SnippetPeReader>
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+static class ExamplePeReader
+{
+    static bool IsAssembly(string path)
+    {
+        var fs = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```

Comment:
```suggestion
        using var fs = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```

Reply:
PEReader automaticaly disposes the underlying stream by default, but having explicit using won't hurt, so i'll apply this.